tests/test_s3_specific.py

```diff
@@ -130,3 +131,24 @@ def _execute_on_subprocess_and_observe(use_threads):
 
         # usually ~15 threads are spun up whe use_threads is True
         assert _execute_on_subprocess_and_observe(use_threads=True) > 10
+
+
+def test_no_sign_request(s3_rig, tmp_path):
+    """Tests that boto3 receives and respects the transfer config
+    when working with a live backend. Does this by observing
+    if the `use_threads` parameter changes to number of threads
+    used by a child process that does a download.
+    """
+    if s3_rig.live_server:
```

Comment:
Maybe instead use
```
@pytest.mark.skipif(os.getenv("USE_LIVE_CLOUD") != "1")
```

Not a big deal. Maybe something we can decide to clean up later since there are other instances of this.

Reply:
For some tests I think we may want different checks if `rig.live_server` is True (e.g., we could have written this to confirm that config was passed to the Mock, but that didn't seem worth it), so I'm more inclined to use checking the rig to determine if we are live or not